lib/phoenix_live_view/html_engine.ex

```diff
@@ -227,9 +243,9 @@ defmodule Phoenix.LiveView.HTMLEngine do
 
   @impl true
   def handle_text(state, meta, text) do
-    %{file: file, indentation: indentation, tokens: tokens, cont: cont} = state
-    {tokens, cont} = HTMLTokenizer.tokenize(text, file, indentation, meta, tokens, cont)
-    %{state | tokens: tokens, cont: cont}
+    %{file: file, indentation: indentation, tokens: tokens, cont: cont, source: source} = state
+    {tokens, cont} = HTMLTokenizer.tokenize(text, file, indentation, meta, tokens, cont, source)
```

Comment:
@josevalim do you think it is ok to send the source here? We need to show the correct snippet.

Reply:
Yes!